plugins/modules/cloud/misc/proxmox.py

```diff
@@ -140,9 +140,11 @@
     type: str
   unprivileged:
     description:
-      - Indicate if the container should be unprivileged
+      - Indicate if the container should be unprivileged.
+      - >
+        The default value for this parameter is C(false) but that is being deprecated
+        and it will be replaced with C(true) in community.general 7.0.0.
```

Comment:
@russoz In this context wouldn't the word _change_ fit better than _replace_?

> The default value will **change** from false to true in community.general 7.0.0.

vs

> The default value of false will be **replaced** with true in community.general 7.0.0.

It sounds more appropriate that "the default value was changed" in case of a value, and "the parameter was replaced with" in case of parameter.

Though English isn't my native language, so I apologize in advance. 😅

Reply:
@shaul75 not mine either, and I like your wording better. This is a bit subjective, so don't sweat it. And you can always come back later and improve the sentence again. ;-)